ui/components/Modals/Information/InfoModal.js

```diff
@@ -400,10 +410,10 @@ const InfoModal_ = React.memo((props) => {
           helpText={
             'Upon submitting your catalog item, an approval flow will be initiated. [Learn More](https://docs.meshery.io/concepts/catalog)'
           }
-          variant="transparent"
+          variant="filled"
         >
           <ActionContainer>
-            <Button
+            <ModalButtonSecondary
```

Comment:
use primarybuttons component it already use the secondary and primary buttons . we don't need to use separately

Reply:
I see, however instead of using them with same name, have a differentiation in name sounds more good to me